packages/datastore/amplify_datastore/pubspec.yaml

```diff
@@ -21,7 +21,7 @@ dependencies:
 
 dev_dependencies:
   amplify_test:
-    path: ../amplify_test
+    path: ../../amplify_test
```

Comment:
I can't comment on the line, but repository should be updated to `https://github.com/aws-amplify/amplify-flutter/tree/next/packages/datastore/amplify_datastore`

Reply:
Whoops, good catch 👍